docs/install/Knative-with-Minishift.md

```diff
@@ -20,7 +20,25 @@ You can find [guides for other platforms here](README.md).
 minishift version
 ```
 
-## Configure and start minishift
+## Automatic Set Up
+
+Once you have `minishift` present on your machine and in your `PATH`,
+you can either follow the manual set up steps below,  or you can run the convenient
+scripts from [the openshift-cloud-functions/knative-operators project](https://github.com/openshift-cloud-functions/knative-operators), 
+which do something similiar, like this:
+
+    git clone https://github.com/openshift-cloud-functions/knative-operators
```

Comment:
I prefer following to keep consistent.

```shell
git clone https://github.com/openshift-cloud-functions/knative-operators
cd knative-operators
./etc/scripts/install-on-minishift.sh 
```

Reply:
@gyliu513 sorry but isn't that exactly how it is? I feel like I must be missing something you are trying to point me to... (If you are a commiter on this project, you can click that Insert a suggestion <Ctrl-G> icon, just before the AA / B / i icons, to directly propose a specific modification.